packages/data/src/export/test/reducer.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * Internal dependencies
+ */
+import reducer from '../reducer';
+import TYPES from '../action-types';
+import { hashExportArgs } from '../utils';
```

Comment:
Could we also use `getResourceName` to create the resource name from the selector + arguments?  Or are there differences that require this to be a unique utility?

Reply:
Oh nice! I was `md5`ing the arguments since report queries can be quite large, but I should be able to use `getResourceName()` instead of directly calling `JSON.stringify()`.

Reply:
Updated in dfd7c67.